sources/logdog/build.rs

```diff
@@ -5,7 +5,20 @@ use std::fs::File;
 use std::io::Write;
 use std::path::PathBuf;
 
+const VARIANT_GROUPS: &[&str] = &["aws-k8s", "aws-ecs"];
+
 fn main() {
+    println!("cargo:rerun-if-env-changed=VARIANT");
+    if let Ok(variant) = env::var("VARIANT") {
+        println!("cargo:rustc-cfg=variant=\"{}\"", variant);
+        for &variant_group in VARIANT_GROUPS {
+            if variant.starts_with(variant_group) {
```

Comment:
I'd rather avoid the idea of variant groups/families until we have time for more thought and design on it.  I think we can do a relatively simple change at the buildsys level that will enable this information for all builds, not just this one, and possibly with a better, more granular mechanism like features/labels for k8s, aws, etc.  Or, at the least, move the decision out of the crate and into a place that already has to care about variant, for example using a data file to list the commands we run and installing a different data file per variant in the os package, or something similar.  In my mind, this feature isn't critical for GA, and waiting is a better option than a one-off, given the complexity involved.